test/integration/Swap.spec.ts

```diff
@@ -0,0 +1,120 @@
+import chai, { expect } from 'chai';
```

Comment:
I think you need to use this `chai` import by adding the line `chai.use(chaiAsPromised);` if you want to use things like `expect.to.eventually.equal`

Reply:
It's interesting that it was working as intended even without the import. Regardless, I added the import as per official `chai-as-promised` usage documentation.